wordle-app/pages/leaderboard.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<v-container class="text-center">
```

Comment:
Page looks great, though a background image may help get rid of some of the blank space around the leader board.

Reply:
Fix it, thanks